benchmarks_gui/CMakeLists.txt

```diff
@@ -41,13 +51,17 @@ include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 link_directories(${Boost_LIBRARY_DIRS})
 link_directories(${catkin_LIBRARY_DIRS})
 
-qt4_wrap_cpp(MOC_FILES include/main_window.h include/trajectory.h include/benchmark_processing_thread.h)
-qt4_wrap_ui(UIC_FILES
-  src/ui/main_window.ui
+set(MOC_FILES include/main_window.h include/trajectory.h include/benchmark_processing_thread.h)
+set(UI_FILES src/ui/main_window.ui
   src/ui/run_benchmark_dialog.ui
   src/ui/robot_loader.ui
   src/ui/bounding_box_goals.ui
 )
+if (UseQt5)
+qt5_wrap_ui(UIC_FILES ${UI_FILES})
+else (UseQt5)
```

Comment:
Yea, I guess its more explicit so you know what if the else corresponds to. If it were me I'd leave it out since its optional, but either way seems fine.